src/components/DataTable/DataTable.tsx

```diff
@@ -99,45 +99,43 @@ const DataTableComponent = memo(
 );
 
 const withDataTableContext = (Component: typeof DataTableComponent) =>
-    memo(
```

Comment:
the fix is not to remove forwardRef.

it is to add ref prop on line 85 and pass it to DataTablePresentationComponent